INSTALL

```diff
@@ -35,6 +35,6 @@
 local-time uses Stefil for testing. To run the Stefil test suite, do
 the following:-
 
-    (ql:quickload :local-time.test)
-    (in-suite test)
```

Comment:
This is ancient. Could you change the instructions to use `(asdf:load-system "local-time")` and `(asdf:test-system "local-time")`?

Reply:
I need to add:
  `(ql:quickload :HU.DWIM.STEFIL)`
in between. Shall I mention it, or do you take quicklisp/asdf taking care of all dependencies for granted?